site/documentation/content/user-guide/amqp-adapter.md

```diff
@@ -451,40 +514,46 @@ Command response sent [outcome: Accepted{}]
 
 ## Downstream Meta Data
 
-The adapter includes the following meta data in the application properties of messages being sent downstream:
+The adapter includes the following meta data in messages being sent downstream:
 
 | Name               | Type      | Description                                                     |
 | :----------------- | :-------- | :-------------------------------------------------------------- |
 | *device_id*        | *string*  | The identifier of the device that the message originates from.  |
 | *orig_adapter*     | *string*  | Contains the adapter's *type name* which can be used by downstream consumers to determine the protocol adapter that the message has been received over. The AMQP adapter's type name is `hono-amqp`. |
 | *orig_address*     | *string*  | Contains the AMQP *target address* that the device has used to send the data. |
 
-The adapter also considers *defaults* registered for the device at either the [tenant]({{< relref "/api/tenant#tenant-information-format" >}})
-or the [device level]({{< relref "/api/device-registration#assert-device-registration" >}}).
+The adapter also considers *defaults* registered for the device at either the
+[tenant]({{< relref "/api/tenant#tenant-information-format" >}}) or the
+[device level]({{< relref "/api/device-registration#assert-device-registration" >}}).
 The values of the default properties are determined as follows:
 
 1. If the message already contains a non-empty property of the same name, the value if unchanged.
-2. Otherwise, if a default property of the same name is defined in the device's registration information, that value is used.
-3. Otherwise, if a default property of the same name is defined for the tenant that the device belongs to, that value is used.
+2. Otherwise, if a default property of the same name is defined in the device's registration information,
+   that value is used.
+3. Otherwise, if a default property of the same name is defined for the tenant that the device belongs to,
+   that value is used.
 
-Note that of the standard AMQP 1.0 message properties only the *content-type* and *ttl* can be set this way to a default value.
+Note that of the standard AMQP 1.0 message properties only the *content-type* and *ttl* can be set this way to a
+default value.
 
 ### Event Message Time-to-live
 
-Events published by devices will usually be persisted by the AMQP Messaging Network in order to support deferred delivery to downstream consumers.
-In most cases the AMQP Messaging Network can be configured with a maximum *time-to-live* to apply to the events so that the events will be removed
-from the persistent store if no consumer has attached to receive the event before the message expires.
+Events published by devices will usually be persisted by the messing infrastructure in order to support deferred
```

Comment:
`messaging infrastructure`
This needs to be fixed in the corresponding chapters in the other protocol adapter guides as well.